src/debugger/BreakpointMap.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+import { Breakpoint } from "./Breakpoint.js";
+
+// Storage for all the breakpoints in one source file
+// Each source file will be associated with one BreakpointStore
+export class BreakpointMap {
```

Comment:
Perhaps rename this to something like "PerFileBreakpointMap".

Reply:
Will do.